support-frontend/assets/components/directDebit/directDebitForm/directDebitForm.tsx

```diff
@@ -116,7 +110,21 @@ function DirectDebitForm(props: PropTypes) {
 				checked={props.accountHolderConfirmation}
 			/>
 
-			{props.phase === 'entry' && <RecaptchaInput id={recaptchaId} />}
+			{props.phase === 'confirmation' && (
+				<RecaptchaInput
+					setRecaptchaToken={props.setRecaptchaToken}
+					expireRecaptchaToken={props.expireRecaptchaToken}
+				/>
+			)}
+
+			{props.formError && (
+				<div className="component-direct-debit-form__error-message-container">
+					<ErrorMessage
+						message={props.formError}
+						svg={<SvgExclamationAlternate />}
+					/>
+				</div>
+			)}
```

Comment:
This was moved above the payment buttons to fix a layout issue (more info in PR description)